flink-core/src/main/java/org/apache/flink/api/java/functions/KeySelector.java

```diff
@@ -36,7 +37,7 @@
 public interface KeySelector<IN, KEY> extends Function, Serializable {
 
 	/**
-	 * User-defined function that extracts the key from an arbitrary object.
+	 * User-defined function that extracts the key from an deterministic object.
```

Comment:
I think this should be `User-defined function that deterministically extracts the key from an object.`

Reply:
Okay. Thanks. I will correct it soon.